### DISCUSSION TITLE
Texlab missing \begin{document}

### DISCUSSION THREAD

**Discussion by noahfraiture** [helix-editor/helix#11449] [Q&A]
Hello, I'm trying to get texlab working but I don't succeed. Currently I can compile via the command line but the lsp doesn't understand the file structure. I have an entry file which is `rapport.tex` and I have being `contenu.tex` which contains the body. The compiler understand and I can build the full pdf, the vscode extension for latex also understand, but texlab in helix give me lsp error which make it pretty unusable. There's nothing the logs. 
Does someone know how to resolve this issue ?

**Comment by cgahr** [comment]
See #9139
